pandas/core/reshape/tile.py

```diff
@@ -417,11 +419,11 @@ def _bins_to_cuts(
         else:
             bins = unique_bins
 
-    side = "left" if right else "right"
+    side: Union[Literal["left"], Literal["right"]] = "left" if right else "right"
```

Comment:
i don't think the typing actually is useful here

Reply:
Actually without this, there was a mypy error https://github.com/pandas-dev/pandas/pull/42448/commits/9814800fa880aa1151eff53cd0d19e329a13dd85. Which was resolved in https://github.com/pandas-dev/pandas/pull/42448/commits/2211736a2ef4cdc607c50b880f519fa6b0377720, where I added this typing.

Reply:
pls rebase on master. i don't think this is necessary

Reply:
rebased, and removed the typing annotation